Clean up Images, Add Syntax Highlighting, Fix Spelling and Grammar (#211)
NuevoFoundation/workshops: content/english/pygame-pong/Wrapup.md

---

# Congratulations! You made it!
**We are excited to see you make it here! Hope you had fun working on your game and learning concepts that will help you make even better games**

## Now, it's your turn!
Use the knowledge you gained from this tutorial to make various interesting changes to the game.

> - Add sets to the game, where each set is the first to 21 points. The player who wins the best of 3 sets will be declared the winner.

Reviewer: ```suggestion
- Add sets to the game, where a player wins a set when they reach 21 points. The player who wins the best of 3 sets will be declared the winner.
```

> - Try adding difficulty level for player to choose. At each difficulty level, the default speed of ball is increased. Can you guess which attribute of the `Ball` object you should change?

Reviewer: ```suggestion
- Try adding difficulty levels for the player to choose. At each difficulty level, the default speed of ball is increased. Can you guess which attribute of the `Ball` object you should change?
```
